src/app/feature/vehicle-detail/vehicle-detail.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="col-lg-12 pt-4">
+  <h2>Vehicle details</h2>
+  <div class="mt-4">
+    <p><b>Capacity:</b> {{selectedVehicleDto?.capacity}}</p>
```

Comment:
Nie lepiej zrobić nullchecka wcześniej  i jeżeli jest null to wyświetlić jakiś komunikat typu "pojazd nie istnieje" czy coś?

Reply:
Zrobiłem